tests/rustdoc-ui/ice-bug-report-url.rs

```diff
@@ -1,4 +1,5 @@
 // compile-flags: -Ztreat-err-as-bug
+// rustc-env:RUSTC_ICE=0
```

Comment:
Do these ui tests use a different default setting for rustc_env? if not, this is unnecessary now

Reply:
Sadly, for rustdoc tests it seems to be needed :-/